core-web/libs/sdk/experiments/src/lib/models.ts

```diff
@@ -0,0 +1,51 @@
+export type Strategy = 'js';
```

Comment:
What is the other one?

Reply:
I will delete it.
In the beginning was to separate the logic between how to run the SDKExperiment (plain js, nextjs, vue, etc) specific logic for each.
I will add it later if I need it.